test/storybook.test.js

```diff
@@ -174,6 +173,10 @@ describe('percy storybook', () => {
     // respond with the preview dom only for the first request
     server.reply('/iframe.html', () => [200, 'text/html', i++ ? storyDOM : previewDOM]);
 
+    // eslint-disable-next-line import/no-extraneous-dependencies
+    let { Percy } = await import('@percy/core');
```

Comment:
lets import on top of the file

Reply:
It's done this way in other tests as well. We don't want spy to propagate